### DISCUSSION TITLE
How Do I Benchmark a Deployed Milvus Cluster?

### DISCUSSION THREAD

**Discussion by RIOTSAKURA** [milvus-io/milvus#27574] [Q&A and General discussion]
I have just started to contact milvus. I have deployed milvus in a cluster. I want to benchmark it now. However, I do not understand the description in the document. I would like to ask if anyone can give me a more detailed answer or some reference cases. Thank you very much.

**Comment by filip-halt** [comment]
Have you taken a look at https://github.com/zilliztech/VectorDBBench ?

**Reply by RIOTSAKURA** [reply]
Okay, I'll try it, thank you.
